uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -846,8 +869,12 @@ define([
        * Initialize user's role
        */
       function initUserRole() {
-        if (this.directiveOptions.leaveRequest &&
-          this.directiveOptions.leaveRequest.contact_id != this.directiveOptions.contactId) {
+        //first condition is if an existing request is opened by user who is not not owner of request
```

Comment:
you mean `isRole`? but `isRole` depends on role being set which is happening here